packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -714,6 +717,10 @@ function ensureRootIsScheduled(root: FiberRoot, currentTime: number) {
 // This is the entry point for every concurrent task, i.e. anything that
 // goes through Scheduler.
 function performConcurrentWorkOnRoot(root) {
+  if (enableProfilerTimer && enableProfilerNestedUpdatePhase) {
+    syncNestedUpdateFlag();
+  }
```

Comment:
I guess I never need this call here, as the nested-update path would always call `performSyncWorkOnRoot`.